src/librustdoc/html/static/js/main.js

```diff
@@ -1219,6 +1219,260 @@ href="https://doc.rust-lang.org/${channel}/rustdoc/how-to-read-rustdoc.html\
     searchState.setup();
 }());
 
+// Hide, show, and resize the sidebar
+//
+// The body class and CSS variable are initially set up in storage.js,
+// but in this file, we implement:
+//
+//   * the show sidebar button, which appears if the sidebar is hidden
+//     and, by clicking on it, will bring it back
+//   * the sidebar resize handle, which appears only on large viewports
+//     with a [fine precision pointer] to allow the user to change
+//     the size of the sidebar
+//
+// [fine precision pointer]: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/pointer
+(function() {
+    // 100 is the size of the logo
+    // don't let the sidebar get smaller than that, or it'll get squished
+    const SIDEBAR_MIN = 100;
```

Comment:
This is a nitpick, not a blocker: I find 100px a bit too small as minimum size:

![image](https://github.com/rust-lang/rust/assets/3050060/d8038acb-dacc-4181-a943-46290a6874ba)

It makes it very difficult to use at this width as we have almost all text cut (like, too much cut).

Reply:
Oh I see. Thanks for the explanations!